appeals/api/src/server/swagger.js

```diff
@@ -449,6 +450,19 @@ const document = {
 			],
 			lpaQuestionnaireId: 1,
 			meetsOrExceedsThresholdOrCriteriaInColumn2: true,
+			neighbouringSiteContacts: [
```

Comment:
I was planning to sync these mocks and the ones used for tests, at some point, if you think it makes sense...

Reply:
The test data is a view of the data before it gets formatted for the response and these are a view of the data after it gets formatted for the response so it might take a bit of work to match them up but it would give a single place for the data.